tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -876,8 +877,8 @@ def crop_center(img, cropx, cropy, cropz):
         return img[startz:endz, starty:endy, startx:endx, :]
 
 
-class AffineTransform(Preprocessor):
-    __provider__ = 'affine_transform'
+class CropProvidedBbox(Preprocessor):
+    __provider__ = 'crop_provided_bbox'
```

Comment:
provided from what?
I am not sure about naming... if it is possible could you please give more descriptive name